lib/xebow/animation_frame.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Xebow.AnimationFrame do
+  @moduledoc false
```

Comment:
Will add :)

Reply:
Added documentation here and updated the other docs throughout. Let me know if something looks out of place since I might have missed something in this second revision.